x-pack/plugin/graph/src/main/java/org/elasticsearch/xpack/graph/rest/action/RestGraphAction.java

```diff
@@ -40,7 +40,6 @@
  */
 public class RestGraphAction extends XPackRestHandler {
     private static final Logger logger = LogManager.getLogger(RestGraphAction.class);
-    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(logger);
```

Comment:
:heart:

Reply:
It will have to temporarily come back in another pull request. 😞